diff --git a/node_modules/@openzeppelin/contracts/governance/utils/IVotes.sol b/node_modules/@openzeppelin/contracts/governance/utils/IVotes.sol
index 7ba012e..80055a2 100644
--- a/node_modules/@openzeppelin/contracts/governance/utils/IVotes.sol
+++ b/node_modules/@openzeppelin/contracts/governance/utils/IVotes.sol
@@ -30,7 +30,7 @@ interface IVotes {
      * @dev Returns the amount of votes that `account` had at a specific moment in the past. If the `clock()` is
      * configured to use block numbers, this will return the value at the end of the corresponding block.
      */
-    function getPastVotes(address account, uint256 timepoint) external view returns (uint256);
+    function getPriorVotes(address account, uint256 timepoint) external view returns (uint256);
 
     /**
      * @dev Returns the total supply of votes available at a specific moment in the past. If the `clock()` is
